Sources/Datadog/RUM/RUMVitals/VitalInfo.swift

```diff
@@ -56,4 +56,31 @@ internal struct VitalInfo {
         }
         return self
     }
+
+    func asFlutterBuildTime() -> RUMViewEvent.View.FlutterBuildTime {
+        return RUMViewEvent.View.FlutterBuildTime(
+            average: meanValue ?? 0.0,
+            max: maxValue ?? 0.0,
+            metricMax: nil,
+            min: minValue ?? 0.0
+        )
+    }
+
+    func asFlutterRasterTime() -> RUMViewEvent.View.FlutterRasterTime {
+        return RUMViewEvent.View.FlutterRasterTime(
+            average: meanValue ?? 0.0,
+            max: maxValue ?? 0.0,
+            metricMax: nil,
+            min: minValue ?? 0.0
+        )
+    }
+
+    func asJsRefreshRate() -> RUMViewEvent.View.JsRefreshRate {
+        return RUMViewEvent.View.JsRefreshRate(
+            average: meanValue ?? 0.0,
+            max: maxValue ?? 0.0,
+            metricMax: 60.0,
+            min: minValue ?? 0.0
+        )
+    }
```

Comment:
I like these convenience methods, but adding them in `VitalInfo` doesn't seem right. The `VitalInfo` is agnostic vital accumulator and here we extend it with vital-specific (even: platform-specific) getters. It will not only add boilerplate for `VitalInfo` type lookup in autocompletion but also require tests coverage as we define new `internal` behaviours.

A better choice to introduce such convenience would be to add private extension in `RUMViewScope.swift` (so the only place where we actually need it):
```swift
// RUMViewScope.swift

// ...

private extension VitalInfo {
   func asFlutterBuildTime() -> RUMViewEvent.View.FlutterBuildTime { /* ... */ }
   func asFlutterRasterTime() -> RUMViewEvent.View.FlutterBuildTime { /* ... */ }
   func asJsRefreshRate() -> RUMViewEvent.View.FlutterBuildTime { /* ... */ }
}
```
This way we only introduce this getters for `VitalInfo` usage in `RUMViewScope.swift` without blowing up the `VitalInfo` namespace in other files. Also, we only introduce `private` behaviours, so we don't need to test it directly (that's our convention).

Eventually, for wider `internal` use, we isolate RUM-related mappings in [`RUMDataModelsMapping`](https://github.com/DataDog/dd-sdk-ios/blob/louiszawadzki/rumm-2514/add-cross-platform-perf-metrics/Sources/Datadog/RUM/DataModels/RUMDataModelsMapping.swift), but I think this is not required for this PR as these getters are scoped only to `RUMViewScope`.